assets/src/blocks/Media/MediaBlock.js

```diff
@@ -36,9 +23,38 @@ export const registerMediaBlock = () => {
     title: __('Media block', 'planet4-blocks-backend'),
     icon: 'format-video',
     category: 'planet4-blocks',
-    attributes,
-    save: MediaFrontend,
+    attributes: {
+      video_title: {
+        type: 'string'
+      },
+      description: {
+        type: 'string'
+      },
+      video_poster_img: {
+        type: 'integer'
+      },
+      embed_html: {
+        type: 'string',
+        default: ''
+      },
+      media_url: {
+        type: 'string'
+      },
+      poster_url: {
+        type: 'string',
+        default: ''
+      },
+    },
+    save: ({ attributes }) => {
+      if (lacksAttributes(attributes)) {
+        return null;
+      }
+
+      return <MediaFrontend { ...attributes } />;
+    },
     edit: MediaEditor,
-    mediaV1,
+    deprecated: [
```

Comment:
I don't think the previous code worked as `mediaV1` was only an array, not an object with the deprecated key.